.github/workflows/go.yml

```diff
@@ -21,3 +21,9 @@ jobs:
         export PATH=$PATH:$(go env GOPATH)/bin
         go build -v .
         GOBIN=$PWD/bin make lint
+
+    - name: Staticcheck
```

Comment:
Maybe we should just use [this](https://staticcheck.io/docs/running-staticcheck/ci/github-actions/)?